Marlin/example_configurations/Hephestos/Configuration.h

```diff
@@ -78,7 +78,7 @@ Here are some standard links for getting your machine calibrated:
 #define CUSTOM_MACHINE_NAME "HEPHESTOS"
 
 // Added for BQ
-#define SOURCE_CODE_URL "http://www.bq.com/gb/downloads-prusa-i3-hephestos.html"
+#define DEFAULT_SOURCE_URL "http://www.bq.com/gb/downloads-prusa-i3-hephestos.html"
```

Comment:
Actually, this is correct. The default source URL is based on the `MOTHERBOARD` selection, then configurations may override it with `SOURCE_CODE_URL`.